medusa/show/naming.py

```diff
@@ -0,0 +1,159 @@
+# coding=utf-8
+
+"""Series naming helpers for selecting results."""
+
+from __future__ import unicode_literals
+
+import fnmatch
+import logging
+import os
+import re
+
+from medusa import app
+from medusa.logger.adapters.style import BraceAdapter
+from medusa.name_parser.parser import InvalidNameException, InvalidShowException, NameParser
+
+from six import string_types
+
+log = BraceAdapter(logging.getLogger(__name__))
+log.logger.addHandler(logging.NullHandler())
+
+
+resultFilters = [
+    '(dir|sub|nfo)fix',
+    '(?<!shomin.)sample',
+    '(dvd)?extras',
+]
+
+if hasattr('General', 'ignore_und_subs') and app.IGNORE_UND_SUBS:
+    resultFilters.append('sub(bed|ed|pack|s)')
+
+if hasattr('General', 'ignored_subs_list') and app.IGNORED_SUBS_LIST:
+    resultFilters.append('(' + app.IGNORED_SUBS_LIST.replace(',', '|') + ')sub(bed|ed|s)?')
+
+
+def contains_words(item, words, strict=True):
```

Comment:
Also going through the function. I can't really justify returning a generator in stead of a list. So that's adding a little complexity without any obvious reason.